plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiCommandLineProcessor.kt

```diff
@@ -42,13 +42,27 @@ class JvmAbiCommandLineProcessor : CommandLineProcessor {
                         "outside of the compilation unit where it's declared.",
                 false,
             )
+
+        val DO_NOT_SORT_MEMBERS_OPTION: CliOption =
```

Comment:
I personally have a preference for flag names without negations, so I would vote for `sortMembers` with a default of true and a comment stating that turning that off reduces stability of the ABI jar.

Reply:
Oh... My preference is flags which are _false by default_... 🙃 For some reason it's easier for me to get a model of a "default" behavior of the system, just looking at the names of the flags, as opposed to names + their default values.

Reply:
!doNotShipIt  🤣

Reply:
More seriously, as long as there is a flag I'm happy! 👍

Reply:
What about `preserveDeclarationsOrder` ? I can rename the flag and make it disable sorting for inner classes as well to be consistent.

Reply:
`preserveDeclarationsOrder` sounds great. `preserveDeclarationOrder` would be even better.